helm_deploy/laa-estimate-eligibility/values/production.yaml

```diff
@@ -61,6 +61,7 @@ featureFlags:
   overrides: NOT_ENABLED
   publicBeta: NOT_ENABLED
   indexProduction: NOT_ENABLED
+  maintenance_mode: NOT_ENABLED
```

Comment:
Ok so this is a good guess, but for consistency, you want to use camelCase not snake_case here. i.e. `maintenanceMode`.

Now, how does that get translated into an appropriate environment variable? Well, for that you need to _also_ add a couple of lines to `helm_deploy/laa-estimate-eligibility/templates/_envs.tpl`, round about line 73